fbw-a380x/src/systems/systems-host/systems/FlightWarningSystem/FwsAbnormalSensed.ts

```diff
@@ -1495,11 +1494,23 @@ export class FwsAbnormalSensed {
       failure: 2,
       sysPage: -1,
     },
+    // 32 LANDING GEAR
+    320800037: {
```

Comment:
Do keep in mind there are two GEAR NOT DOWN warnings. One which is able to be silenced and another one which is not. You can copy it from my #8958 pr if you want as I ported it there from the A32NX or keep it as is so it gets overwritten later.

Reply:
Ahhh, that's where it was. I was pretty sure I once saw it triggered during flight, but then I couldn't find it on master. Makes sense to handle the whole chapter at once, I'll delete it from my PR